README.md

```diff
@@ -195,6 +195,24 @@ Install custom reporters as a development dependency and you can use the `--repo
 nyc report --reporter=<custom-reporter-name>
 ```
 
+## Setting the project root directory
+
+nyc runs a lot of file system operations relative to the project root directory.
+During startup nyc will look for the *default* project root directory.
+The *default* project root directory is the first directory found that contains a `package.json` file when searching from the current working directory up.
+If nyc fails to find a directory containing a `package.json` file, it will use current working directory.
+You can change the project root directory with the `--cwd` option.
+
+nyc uses the project root directory when:
+ * looking for source files to cover
+ * creating globs for include and exclude rules during file selection
+ * loading custom require hooks from the `require` array
+
+nyc may create artefact directories within the project root, such as:
+  * the report directory, `<project-root>/coverage`
+  * the cache directory, `<project-root>/node_modules/.cache/nyc`
```

Comment:
in the tool I've been working on called `c8`, we consolidated on the `coverage` folder, rather than having a `.nyc_output` folder, we might want to consider this in future nyc versions.

Reply:
Do we want to do this for 14.0.0?  This would be a bit of a breaking change but I think it's a good idea.  Do you think `<project-root>/coverage/nyc-tmp`? in place of the `.nyc_output` folder?